### PR TITLE
docs: add ui-plugin-index

### PR DESCRIPTION
**Pull request by WuJun2016**
doc issue: https://github.com/harvester/harvester/issues/4328

### CODE REVIEW

File: docs/advanced/settings.md [hunk 1]
```diff
@@ -331,6 +331,18 @@ Default: `https://releases.rancher.com/harvester-ui/dashboard/latest/index.html`
 https://your.static.dashboard-ui/index.html
 ```
 
+## `ui-plugin-index`
+
+This setting allows you to configure the JS address for the harvester plugin (When access Harvester from Rancher).
```
Comment (vickyhella) [inline docs/advanced/settings.md:336]:
```suggestion
This setting allows you to configure the JS address for the Harvester plugin (when accessing Harvester from Rancher).
```

File: i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md [hunk 2]
```diff
@@ -323,6 +323,18 @@ IP 范围格式是 IPv4 CIDR，而且是集群节点数的 4 倍。
 https://your.static.dashboard-ui/index.html
 ```
 
+## `ui-plugin-index`
+
+为 harvester 配置插件的 JS 地址 (从rancher中访问harvester时使用).
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md:328]:
```suggestion
为 Harvester 插件配置 JS 地址 (从 Rancher 中访问 Harvester 时使用)。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md [hunk 3]
```diff
@@ -323,6 +323,18 @@ IP 范围格式是 IPv4 CIDR，而且是集群节点数的 4 倍。
...
+
+Default: `https://releases.rancher.com/harvester-ui/plugin/harvester-latest/harvester-latest.umd.min.js`
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md:330]:
```suggestion
默认值：`https://releases.rancher.com/harvester-ui/plugin/harvester-latest/harvester-latest.umd.min.js`
```

File: i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md [hunk 4]
```diff
@@ -323,6 +323,18 @@ IP 范围格式是 IPv4 CIDR，而且是集群节点数的 4 倍。
...
+
+#### Example
```
Comment (vickyhella) [inline i18n/zh/docusaurus-plugin-content-docs/current/advanced/settings.md:332]:
```suggestion
#### 示例
```

**Review by vickyhella — COMMENTED** [review]
Only some nits

**Review by vickyhella — APPROVED** [review]
LGTM!

**Review by LucasSaintarbor — APPROVED** [review]
+1 LGTM

**Comment by LucasSaintarbor** [comment]
@WuJun2016 Is this PR associated to a specific /harvester issue?

**Comment by WuJun2016** [comment]
> @WuJun2016 Is this PR associated to a specific /harvester issue?

https://github.com/harvester/harvester/issues/4328

File: docs/advanced/settings.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (WuJun2016) [reply docs/advanced/settings.md]: I think both translations should be OK, but I may need your advice on which one is more suitable

**Pull request merged**